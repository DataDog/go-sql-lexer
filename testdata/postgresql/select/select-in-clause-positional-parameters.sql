SELECT * FROM orders WHERE status IN ($1, $2, $3);
