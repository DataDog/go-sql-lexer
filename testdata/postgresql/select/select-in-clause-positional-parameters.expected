[
    {
        "expected": "SELECT * FROM orders WHERE status IN ( ? )"
    },
    {
        "obfuscator_config": {
            "replace_positional_parameter": false
        },
        "expected": "SELECT * FROM orders WHERE status IN ( $1, $2, $3 )"
    }
]